src/Illuminate/Contracts/Events/Dispatcher.php

```diff
@@ -48,6 +48,14 @@ public function until($event, $payload = []);
      */
     public function dispatch($event, $payload = [], $halt = false);
 
+    /**
+     * Dispatch multiple events and call their listeners.
+     *
+     * @param  array<int|string, mixed>  $events
+     * @return void
+     */
+    public function dispatchMultiple($events);
```

Comment:
I'd like to propose renaming this to `dispatchMany` as the `xMany` convention is used throughout the framework e.g. `createMany`

Reply:
I think you commented just as I pushed this haha